pkg/cache/context.go

```diff
@@ -935,11 +935,11 @@ func (ctx *Context) PublishEvents(eventRecords []*si.EventRecord) {
 		for _, record := range eventRecords {
 			switch record.Type {
 			case si.EventRecord_REQUEST:
-				taskID := record.ObjectID
-				appID := record.GroupID
+				appID := record.ObjectID
+				taskID := record.ReferenceID
 				if task := ctx.getTask(appID, taskID); task != nil {
 					events.GetRecorder().Eventf(task.GetTaskPod().DeepCopy(), nil,
-						v1.EventTypeNormal, record.Reason, record.Reason, record.Message)
+						v1.EventTypeNormal, "", "", record.Message)
```

Comment:
Will these two record.Reason paras will be removed for Eventf() function?

Reply:
I don't know, we might need a replacement. We no longer have the "reason" field in the message. I'll talk to Wilfred about this. For this PR, it's OK to leave it empty and we'll see what we do.

Reply:
Sure, it makes sense, it's ok for this PR!